source/mqtt5_client.c

```diff
@@ -3517,6 +3527,8 @@ napi_value aws_napi_mqtt5_client_close(napi_env env, napi_callback_info info) {
         return NULL;
     }
 
+    s_aws_mqtt5_release_callbacks(binding);
```

Comment:
This is not safe to do here.  We need to dig a lot deeper to find out what's going on here to cause problems.